Assets/_Scripts/UI/Popups/PerformanceInfoPopup.cs

```diff
@@ -18,6 +18,11 @@ public class PerformanceInfoPopup : MonoBehaviour
         public void SetPerformanceCardInfo(PerformanceDataSO data)
         {
             _performanceData = data;
+            TrackDataSO trackData = data.trackData;
+            performanceName.text = trackData.questName;
+            composer.text = trackData.composerName;
+            info.text = trackData.info;
+            // performanceTime.text =  TODO later as data comes from night not performance
```

Comment:
What is meant by "performance time"?

Reply:
it's meant to be the date/night info we show on the UI night select card